frontend/src/lib/components/ResizableTable/TableConfig.scss

```diff
@@ -24,13 +23,65 @@
         }
 
         &.selected {
-            background-color: var(--primary-hover);
+            background-color: #eef2ff;
         }
 
         .property-key-info {
             flex-grow: 0;
             overflow: hidden;
             text-overflow: ellipsis;
         }
+
+        .item-label {
+            padding: 0;
+        }
+    }
+
+    .main-content {
+        margin-bottom: 8px;
+
+        .lists {
+            background-color: var(--bg-side);
+            border-radius: 4px;
+            padding: 8px;
```

Comment:
Here and elsewhere: `8px` -> `0.5rem`

https://storybook.posthog.net/?path=/story/lemon-ui-overview--page#lemon-principles